CHANGELOG.md

```diff
@@ -1,31 +1,37 @@
+# Changelog
```

Comment:
Most of the changes to this file came along when I linted markdown files without thinking. They do not need to be part of this PR and could be reverted.